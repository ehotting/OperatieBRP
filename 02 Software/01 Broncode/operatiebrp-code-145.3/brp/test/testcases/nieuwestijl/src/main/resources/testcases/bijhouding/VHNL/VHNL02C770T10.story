Meta:
@auteur                 tjlee
@status                 Klaar
@regels                 R2177
@sleutelwoorden         voltrekkingHuwelijkInNederland,TjieWah,VHNL02C770T10
@usecase                UCS-BY.HG

Narrative:
R2177 Huwelijk en geregistreerd partnerschap, voltrekking huwelijk in Nederland, actie registratieAanvangHuwelijkGeregistreerdPartnerschap

Scenario: R2177 Datum geboorte ligt in de toekomst
          LT: VHNL02C770T10



Given alle personen zijn verwijderd
Given enkel initiele vulling uit bestand /LO3PL/VHNL02C770-sandy.xls
Given enkel initiele vulling uit bestand /LO3PL/VHNL02C770-danny.xls

When voer een bijhouding uit VHNL02C770T10.xml namens partij 'Gemeente BRP 1'

Then heeft het antwoordbericht verwerking Foutief
And is het antwoordbericht gelijk aan /testcases/bijhouding/VHNL/expected/VHNL02C770T10.xml voor expressie //brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R

Then is in de database de persoon met bsn 558376617 niet als PARTNER betrokken bij een HUWELIJK
Then is in de database de persoon met bsn 156960849 niet als PARTNER betrokken bij een HUWELIJK

Then lees persoon met anummer 1686982546 uit database en vergelijk met expected VHNL02C770T10-persoon1.xml
