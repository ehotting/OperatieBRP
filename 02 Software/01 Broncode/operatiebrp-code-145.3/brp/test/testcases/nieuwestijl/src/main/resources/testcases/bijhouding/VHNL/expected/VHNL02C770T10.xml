<?xml version="1.0" encoding="UTF-8"?><brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R xmlns:brp="http://www.bzk.nl/brp/brp0200">
    <brp:stuurgegevens>
        <brp:zendendePartij>199901</brp:zendendePartij>
        <brp:zendendeSysteem>BRP</brp:zendendeSysteem>
      <brp:referentienummer>*</brp:referentienummer>
        <brp:crossReferentienummer>00000000-0000-0000-0042-200000000104</brp:crossReferentienummer>
      <brp:tijdstipVerzending>*</brp:tijdstipVerzending>
    </brp:stuurgegevens>
    <brp:resultaat>
        <brp:verwerking>Foutief</brp:verwerking>
        <brp:hoogsteMeldingsniveau>Fout</brp:hoogsteMeldingsniveau>
    </brp:resultaat>
    <brp:meldingen>
        <brp:melding brp:objecttype="Melding" brp:referentieID="comid.voltrekkingHuwelijkInNederland.partner2.persoon.geboorte">
            <brp:regelCode>R2177</brp:regelCode>
            <brp:soortNaam>Fout</brp:soortNaam>
            <brp:melding>Datum geboorte mag niet in de toekomst liggen.</brp:melding>
        </brp:melding>
    </brp:meldingen>
    <brp:voltrekkingHuwelijkInNederland brp:objecttype="AdministratieveHandeling">
        <brp:partijCode>507013</brp:partijCode>
      <brp:tijdstipRegistratie>*</brp:tijdstipRegistratie>
    </brp:voltrekkingHuwelijkInNederland>
</brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R>
